.eslintrc.yml

```diff
@@ -15,18 +16,8 @@ plugins:
 root: true
 
 rules:
-  security/detect-buffer-noassert: warn
-  security/detect-child-process: warn
-  security/detect-disable-mustache-escape: warn
-  security/detect-eval-with-expression: warn
-  security/detect-new-buffer: warn
-  security/detect-no-csrf-before-method-override: warn
-  security/detect-non-literal-fs-filename: warn
-  security/detect-non-literal-regexp: warn
-  security/detect-non-literal-require: warn
-  security/detect-possible-timing-attacks: warn
-  security/detect-pseudoRandomBytes: warn
-  security/detect-unsafe-regex: warn
+  security/detect-non-literal-fs-filename: off
+  security/detect-object-injection: off
```

Comment:
Disable these 2 rules since the non-literal filesystem filenames are only for local development, and the other rule is kinda noisy.